Fixed #35863 -- replaced bold text with heading level 3 (#18760)
django/django: docs/internals/contributing/new-contributors.txt

Start with these steps to discover Django's development process.

> Triage tickets
> -----------------

Reviewer: ```suggestion
Triage tickets
--------------
```

  If an `unreviewed ticket`_ reports a bug, try and reproduce it. If you

>   can reproduce it and it seems valid, make a note that you confirmed the bug

Author: UGGGGHHHHH.... How did I miss that?!

  behavior, even if you don't fix the bug itself. See more at
  :ref:`how-can-i-help-with-triaging`

> Look for tickets that are accepted and review patches to build familiarity with the codebase and the process
> ------------------------------------------------------------------------------------------------------------
> 
>   Mark the appropriate flags if a patch needs docs or tests. Look through the
>   changes a patch makes, and keep an eye out for syntax that is incompatible

Reviewer: I think this is too long for a header, so let's move some of this into the description
To get you started, something like:
```suggestion
Review patches of accepted tickets
----------------------------------

This will help you to build familiarity with the codebase and processes. Mark
the appropriate flags if a patch needs docs or tests. Look through the changes
a patch makes, and keep an eye out for syntax that is incompatible with older
```

Author: Thanks for the suggestion. I didn't even think to make any content changes.

As a newcomer on a large project, it's easy to experience frustration. Here's
some advice to make your work on Django more useful and rewarding.

> Pick a subject area that you care about, that you are familiar with, or that you want to learn about
> ----------------------------------------------------------------------------------------------------
> 
>   You don't already have to be an expert on the area you want to work on; you
>   become an expert through your ongoing contributions to the code.

Reviewer: Similar suggestion with the long headers
```suggestion
Pick a subject area
-------------------

This should be something that you care about, that you are familiar with, or
that you want to learn about. You don't already have to be an expert on the
area you want to work on; you become an expert through your ongoing
contributions to the code.
```

  recently involved in a discussion, then a ticket may not have the support
  required to get into Django.

Start small
-----------

  It's easier to get feedback on a little issue than on a big one. See the
  `easy pickings`_.

If you're going to engage in a big task, make sure that your idea has support first
-----------------------------------------------------------------------------------

  This means getting someone else to confirm that a bug is real before you fix
  the issue, and ensuring that there's consensus on a proposed feature before
  you go implementing it.

Be bold! Leave feedback!
------------------------

  Sometimes it can be scary to put your opinion out to the world and say "this
  ticket is correct" or "this patch needs work", but it's the only way the
  project moves forward. The contributions of the broad Django community
  ultimately have a much greater impact than that of any one person. We can't
  do it without **you**!

> Err on the side of caution when marking things Ready For Check-in

Author: @sarahboyce 
This doesn't look long in code but on the web page it is on two lines. Should we also shorten this heading as well?

![heading2lines](https://github.com/user-attachments/assets/886a10c2-8aff-4baf-aaff-c1da93989ae2)

Reviewer: I agree, maybe something like `Be careful setting tickets as "Ready For Check-in"`?

Also, I know you pushed up changes but they look to be lost, can you push up again? :pray:
